domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressController.kt

```diff
@@ -111,7 +111,8 @@ class ExplorationProgressController @Inject constructor(
   private val loggingIdentifierController: LoggingIdentifierController,
   private val profileManagementController: ProfileManagementController,
   private val learnerAnalyticsLogger: LearnerAnalyticsLogger,
-  @BackgroundDispatcher private val backgroundCoroutineDispatcher: CoroutineDispatcher
+  @BackgroundDispatcher private val backgroundCoroutineDispatcher: CoroutineDispatcher,
+  private val explorationProgressListener: ExplorationProgressListener
```

Comment:
```suggestion
  private val explorationProgressListeners: Set<explorationProgressListener>
```

Reply:
Got it, thanks!
An interesting thing I encountered was the need to annotate the type argument here with @JvmSuppressWildcards as the [documentation](https://dagger.dev/dev-guide/multibindings.html) warns.